model/src/main/java/org/projectnessie/model/LogResponse.java

```diff
@@ -49,6 +51,9 @@ static ImmutableLogEntry.Builder builder() {
     @NotNull
     CommitMeta getCommitMeta();
 
+    @JsonInclude(Include.NON_EMPTY)
```

Comment:
This one breaks compatibility w/ older clients, when the client retrieves a commit-log-entry that has an additional parent (a merge-commit).

Reply:
Can we make JSON parsing more lenient on the client?